students/JamesTraub/lesson03/list_lab.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+# Lesson 3 - Series 1
+
+fruit = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print("Create and display a list of fruit.")
+print(fruit)
+print()
+
+inp_fruit = input('Please enter a name of a fruit to add to the end of list. ')
+fruit.append(inp_fruit)
+print(fruit)
+print()
+
+fruit_number = int(input('Please enter the number that correspends with one fruit in the list. '))
+for i, nfruit in enumerate(fruit):
+    if fruit_number == (i + 1):
+        print(fruit_number, nfruit)
+print()
+
+add_fruit2 = []
+inp_fruit2 = input('Please enter a fruit to add to the beginning of the list. ')
+fruit = [inp_fruit2,] + fruit
+print(fruit)
+print()
+
+add_fruit3 = []
+add_fruit = input('Please enter a fruit to add to the beginning of the list. ')
+add_fruit3.insert(0, add_fruit)
+fruit = add_fruit3 + fruit
```

Comment:
this should just be: `fruit.insert(0, add_fruit)` no need to create additional temp list